language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/references/ReferencesTreeVisitor.java

```diff
@@ -101,11 +103,15 @@ public ReferencesTreeVisitor(LSServiceOperationContext context) {
 
     @Override
     public void visit(BLangPackage pkgNode) {
-        if (pkgNode.topLevelNodes.isEmpty()) {
+        List<TopLevelNode> topLevelNodes = new ArrayList<>(pkgNode.topLevelNodes);
+        if (!pkgNode.testablePkgs.isEmpty()) {
```

Comment:
Instead of checking if the testable package list is empty shall we use the method already used to check if a testable package exist or not 'containsTestablePkg()' given in  https://github.com/ballerina-platform/ballerina-lang/blob/master/compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/BLangPackage.java#L226?